doc/examples/reconst_dki.py

```diff
@@ -238,7 +238,28 @@
 The standard kurtosis statistics can be computed from the DiffusinKurtosisFit
 instance as the mean kurtosis (MK), the axial kurtosis (AD) and the radial
 kurtosis (RK).
+"""
+
+MK = dkifit.mk
+RK = dkifit.rk
+AK = dkifit.ak
+
+fig3, ax = plt.subplots(1, 3, figsize=(12, 6),
+                        subplot_kw={'xticks': [], 'yticks': []})
+
+fig3.subplots_adjust(hspace=0.3, wspace=0.05)
 
+ax.flat[0].imshow(MK[:, :, axial_middle], cmap='gray')
+ax.flat[0].set_title('MK')
+ax.flat[1].imshow(RK[:, :, axial_middle], cmap='gray')
+ax.flat[1].set_title('RK')
+ax.flat[2].imshow(AK[:, :, axial_middle], cmap='gray')
+ax.flat[2].set_title('AK')
+
+plt.show()
+fig3.savefig('Kurtosis_tensor_standard_measures.png')
```

Comment:
You need to add the `.. figure` directive in the text block below for this figure to be included in the webpage